docs/source/intro.rst

```diff
@@ -16,8 +16,16 @@ The preferred way of importing pynucastro is as follows:
 The main classes are:
 
 * :func:`Nucleus <pynucastro.rates.rate.Nucleus>`: This is a single
-  nucleus.  It knows its proton number, Z, neutron number, N, and
-  weight, A.
+  nucleus.  It knows its proton number, ``Z``, neutron number, ``N``,
```

Comment:
What happens if the binding energy is None?

Reply:
I don't think it can be None.  But if it is None, then anything that accesses this will need to deal with it.

It hasn't come up in any of the nets we've generated previously.